src/views/RecIM/components/Helpers/index.js

```diff
@@ -1,11 +1,35 @@
 import { DateTime } from 'luxon';
 
 const standardDate = (date, includeTime) => {
+  if (!DateTime.isDateTime(date)) date = DateTime.fromISO(date);
   let formattedDate = date.monthShort + ' ' + date.day;
   if (includeTime) {
     formattedDate += ' ' + date.toLocaleString(DateTime.TIME_SIMPLE);
   }
   return formattedDate;
 };
 
-export { standardDate };
+const standardDateTimeRange = (startDateTime, endDateTime) => {
+  if (!DateTime.isDateTime(startDateTime)) startDateTime = DateTime.fromISO(startDateTime);
+  if (!DateTime.isDateTime(endDateTime)) endDateTime = DateTime.fromISO(endDateTime);
+
+  // Jan 1 3:00pm - 6:00pm
+  let isOneDay =
+    startDateTime.toLocaleString(DateTime.DATE_FULL) ===
+    endDateTime.toLocaleString(DateTime.DATE_FULL);
+  if (isOneDay) {
+    let endTime = startDateTime.toLocaleString(DateTime.TIME_SIMPLE).replace(':00', '');
+    return `${standardDate(startDateTime, true).replace(':00', '')} - ${endTime}`;
+  }
+  // Jan 1 3:00pm - Jan 31 3:00pm
+  let isOneYear = startDateTime.year === endDateTime.year;
+  if (isOneYear) {
+    return `${standardDate(startDateTime)} - ${standardDate(endDateTime)}`;
+  }
+  // Jan 1 2023 3:00pm - Jan 1 2024 3:00pm
+  return `${startDateTime.toLocaleString(DateTime.DATE_MED)} - ${endDateTime.toLocaleString(
+    DateTime.DATE_MED,
```

Comment:
you're going to have to explain this to me in person

Reply:
ah, okay but what is the usecase for over a year, we do that in testing but there should never be a rec-im activity that lasts more than a couple months at max

Reply:
I might call this `formatDateTimeRange(...)` to imply that it is returning a string format of the date/s passed in. I think that makes a little bit more sense to a developer without context.

Reply:
> ah, okay but what is the usecase for over a year? we do that in testing but there should never be a rec-im activity that lasts more than a couple months at max

I'd push back on this @amos-cha because I believe this is a generic function that promises to handle date ranges and return a proper formatted string. It should not be tied in with the logic of the rest of the application unless it has a more tailored function.